### PR TITLE
(docs) Add references in Readme

### PR DESCRIPTION
**Pull request by daniel-hc**
## What changed?
References to Apache Kafka and Typescript were added to the Readme file.

## Why is this change needed?
This PR is intended to be a template for further contributions. What I propose here es to commit changes to a separate branch and require at least 1 approval to merge into `master`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ## Kafka App
```
Comment (daniel-hc) [inline README.md:1]: Why is this Title defined as a second level header? I mean, why not start with the first level?

```md
# Kafka App
```

**Review by daniel-hc — COMMENTED** [review]
@RodolfoFrias WDYT of changing title to level 1 header?

File: README.md (hunk 1 above)
Comment (daniel-hc) [inline README.md]:
```suggestion
# Kafka App
```

**Comment by daniel-hc** [comment]
What if we add more details in the Readme file like: What is this app for? How does it implement it? How to install/use it? Dependencies, the target public, and any other insight that make this Readme more useful for other developers before jumping into the code.

Usually, we write instructions on how to install the app, run tests, deploy changes, and so on.

**Comment by RodolfoFrias** [comment]
I've addressed some of the comments, I still have to update the readme file, coming soon...

**Review by RodolfoFrias — APPROVED** [review]

**Pull request merged**